src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandlerTest.java

```diff
@@ -35,110 +53,172 @@ public class AdjournCaseAboutToSubmitHandlerTest {
+                .hearingRoute(HearingRoute.GAPS)
+                .build())
             .build();
         when(caseDetails.getCaseData()).thenReturn(sscsCaseData);
     }
 
+    @After
+    public void tearDown() throws Exception {
+        autoCloseable.close();
+    }
+
+    @DisplayName("Given a non adjourn case event, then return false")
     @Test
-    public void givenANonAdjournCaseEvent_thenReturnFalse() {
+    public void testNonAdjournCase() {
         when(callback.getEvent()).thenReturn(EventType.APPEAL_RECEIVED);
-        assertFalse(handler.canHandle(ABOUT_TO_SUBMIT, callback));
+        assertThat(handler.canHandle(ABOUT_TO_SUBMIT, callback)).isFalse();
     }
 
+    @DisplayName("Given draft adjournment notice already exists on case, then overwrite existing draft")
     @Test
-    public void givenDraftAdjournmentNoticeAlreadyExistsOnCase_thenOverwriteExistingDraft() {
+    public void testOverwriteExistingDraft() {
         SscsDocument doc = SscsDocument.builder().value(SscsDocumentDetails.builder().documentFileName("oldDraft.doc").documentType(DRAFT_ADJOURNMENT_NOTICE.getValue()).build()).build();
         List<SscsDocument> docs = new ArrayList<>();
         docs.add(doc);
         callback.getCaseDetails().getCaseData().setSscsDocument(docs);
 
-        PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
+        handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
 
-        assertEquals(1, response.getData().getSscsDocument().size());
-        assertEquals((String.format("Draft Adjournment Notice generated on %s.pdf", LocalDate.now().format(DateTimeFormatter.ofPattern("dd-MM-YYYY")))), response.getData().getSscsDocument().get(0).getValue().getDocumentFileName());
+        verify(previewDocumentService, times(1)).writePreviewDocumentToSscsDocument(
+            sscsCaseData, DRAFT_ADJOURNMENT_NOTICE, null);
     }
 
+    @DisplayName("Given an adjournment event with language interpreter required and case has existing interpreter, "
+        + "then overwrite existing interpreter in hearing options")
     @Test
-    public void givenAnAdjournmentEventWithLanguageInterpreterRequiredAndCaseHasExistingInterpreter_thenOverwriteExistingInterpreterInHearingOptions() {
-        callback.getCaseDetails().getCaseData().setAdjournCaseInterpreterRequired("Yes");
+    public void testOverwriteExistingInterpreter() {
+        callback.getCaseDetails().getCaseData().setAdjournCaseInterpreterRequired(YES.getValue());
         callback.getCaseDetails().getCaseData().setAdjournCaseInterpreterLanguage("Spanish");
-        callback.getCaseDetails().getCaseData().getAppeal().setHearingOptions(HearingOptions.builder().languageInterpreter("No").languages("French").build());
+        callback.getCaseDetails().getCaseData().getAppeal().setHearingOptions(HearingOptions.builder()
+            .languageInterpreter(NO.getValue())
+            .languages("French")
```

Comment:
Could enum or have languages as constants for reuse.

Reply:
sure